cl_sii/rcv/data_models.py

```diff
@@ -214,79 +227,109 @@ def as_dte_data_l2(self) -> cl_sii.dte.data_models.DteDataL2:
         return dte_data
 
 
-@dataclasses.dataclass(frozen=True)
+@pydantic.dataclasses.dataclass(
+    frozen=True,
+    config=type('Config', (), dict(
+        arbitrary_types_allowed=True,
+    ))
+)
 class RvDetalleEntry(RcvDetalleEntry):
 
     """
     Entry of the "detalle" of an RV ("Registro de Ventas").
     """
 
+    ###########################################################################
+    # constants
+    ###########################################################################
+
+    DATETIME_FIELDS_TZ = SII_OFFICIAL_TZ
+
     RCV_KIND = RcvKind.VENTAS
     RC_ESTADO_CONTABLE = None
 
     # TODO: docstring
     # TODO: can it be None? What happens for those "tipo docto" that do not have a receptor?
-    receptor_razon_social: str = dc_field()
+    receptor_razon_social: str
 
     # TODO: docstring
     # note: must be timezone-aware.
-    fecha_acuse_dt: Optional[datetime] = dc_field()
+    fecha_acuse_dt: Optional[datetime] = None
 
     # TODO: docstring
     # note: must be timezone-aware.
-    fecha_reclamo_dt: Optional[datetime] = dc_field()
-
-    def __post_init__(self) -> None:
-        super().__post_init__()
-
-        if not isinstance(self.receptor_razon_social, str):
-            raise TypeError("Inappropriate type of 'receptor_razon_social'.")
-        cl_sii.dte.data_models.validate_contribuyente_razon_social(self.receptor_razon_social)
-
-        if self.fecha_acuse_dt is not None:
-            if not isinstance(self.fecha_acuse_dt, datetime):
-                raise TypeError("Inappropriate type of 'fecha_acuse_dt'.")
-            tz_utils.validate_dt_tz(self.fecha_acuse_dt, SII_OFFICIAL_TZ)
-
-        if self.fecha_reclamo_dt is not None:
-            if not isinstance(self.fecha_reclamo_dt, datetime):
-                raise TypeError("Inappropriate type of 'fecha_reclamo_dt'.")
-            tz_utils.validate_dt_tz(self.fecha_reclamo_dt, SII_OFFICIAL_TZ)
+    fecha_reclamo_dt: Optional[datetime] = None
```

Comment:
See review comment for field `fecha_acuse_dt`.

```suggestion
    fecha_reclamo_dt: Optional[datetime]
```